apps/_compability_app/services/UserService.js

```diff
@@ -0,0 +1,48 @@
+import { injectable } from "inversify";
+import { globalApolloClient } from "@core/next/apollo";
+import { USER_QUERY } from "@core/next/auth";
```

Comment:
It's better to create your project GQL query and pass it here and inside `useAuth`